src/core/Modal/__tests__/Modal.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { KuberaThemeProvider } from '../../../theme';
+import Modal from '../Modal';
+
+describe('Modal Component', () => {
+  it('Renders', () => {
+    const { findByTitle } = render(
+      <KuberaThemeProvider platform="kubera-chaos">
+        <Modal open={true} onClose={() => console.log('clicked')} hasCloseBtn>
+          <>Modal</>
```

Comment:
Modal inside a modal?

Reply:
The { Children } will need a jsx type in return, so passing just modal will not work so just fragment and them passing Modal text inside it.